media-radio/KochMorse/KochMorse-3.4.0.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6	# cmake-utils.eclass does not support EAPI 7
+
+inherit cmake-utils
+
+DESCRIPTION="Morse-code tutor using the Koch method"
+HOMEPAGE="https://github.com/hmatuschek/kochmorse"
+SRC_URI="https://github.com/hmatuschek/kochmorse/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="dev-qt/linguist-tools
+	dev-qt/qtmultimedia
+	dev-util/cmake[qt5]"
+RDEPEND="dev-qt/qtmultimedia"
+
+S="${WORKDIR}/kochmorse-${PV}"
+
+PATCHES=( "${FILESDIR}/desktop.patch" )
```

Comment:
This patch isn't very descriptive, and is (in my opinion) easier to fix with one line of sed inside src_prepare. Report it to upstream to get it fixed by them for the next release, too :)